CHANGELOG.md

```diff
@@ -3,6 +3,7 @@ Change log
 
 # v2.0.0-beta.7 - 2016-12-05
 
+* Update tests/start.sh - Enable AutoHaT support for Qemu 32 bit images [Horia]
```

Comment:
This need to be added on top. You can't add something to a release already done.